plugins/lando-recipes/lamp/lamp.js

```diff
@@ -336,6 +337,36 @@ module.exports = function(lando) {
     build.services = services(config);
     build.tooling = tooling(config);
 
+    // Determine the service to run cli things on
+    var unsupportedCli = (config.php === '5.3' || config.php === 5.3);
```

Comment:
@serundeputy this stuff is pretty pantheon specific. whats the thinking around moving this over to lamp??

Reply:
@pirog if a user is using lemp and php 5.3 it will spin up a php appserver_cli container.  WE don't have to use the pantheon image for it; do you have a recommendation for a better image here?

Reply:
@serundeputy composer works in php 5.3 so i assumed we'd just run it in the appserver without needed a cli container. We only use the cli container on pantheon in the 5.3 use case because terminus requires 5.5+

Reply:
gotchya; I'm moving this back to the Pantheon recipe.